beets/ui/commands.py

```diff
@@ -1666,6 +1666,7 @@ def print_completion(*args):
     u'/usr/share/local/bash-completion/bash_completion',
     u'/opt/local/share/bash-completion/bash_completion',  # SmartOS
     u'/usr/local/etc/bash_completion',  # Homebrew
+    u'/usr/local/share/bash-completion/bash_completion',  # Homebrew bash-completion2
```

Comment:
Hmm, the line is longer than 79 characters which is causing our flake8 test to fail. Maybe move all of the comments to the line before? e.g.

```python
BASH_COMPLETION_PATHS = map(syspath, [
    # General *nix
    u'/etc/bash_completion',
    u'/usr/share/bash-completion/bash_completion',
    u'/usr/share/local/bash-completion/bash_completion',
    # SmartOS
    u'/opt/local/share/bash-completion/bash_completion',
    # Homebrew bash-completion
    u'/usr/local/etc/bash_completion',
    # Homebrew bash-completion2
    u'/usr/local/share/bash-completion/bash_completion',  
])
```